xapian-applications/omega/worker.cc

```diff
@@ -25,15 +25,16 @@
 #include "worker.h"
 #include "worker_comms.h"
 
-#include "pkglibbindir.h"
 #include <csignal>
 #include <cstring>
 #include <cerrno>
-#include <iostream>
-#include "safefcntl.h"
-#include "safeunistd.h"
 #include <sys/socket.h>
 #include <sys/wait.h>
+
+#include "parseint.h"
+#include "pkglibbindir.h"
+#include "safefcntl.h"
+#include "safeunistd.h"
 #include "closefrom.h"
 #include "freemem.h"
```

Comment:
As above, I don't think the `"safeXXX.h"` headers should be here.

Moving `"pkglibbindir.h"` to this block makes sense, and adding `"parseint.h"` here is good too, but it would be better to maintain the sorted order of the headers in this block of headers.